src/Spatial/Units/Angle.cs

```diff
@@ -311,36 +311,39 @@ public static Angle ReadFrom(XmlReader reader)
             return reader.ReadElementAs<Angle>();
         }
 
-        /// <inheritdoc />
+        /// <summary>
+        /// Returns a string representation of an Angle according to the default representation provided by <see cref="AngleFormatProvider"/>
+        /// </summary>
+        /// <returns>The string representation of this instance.</returns>
         public override string ToString()
         {
-            return this.ToString(null, NumberFormatInfo.CurrentInfo);
+            return this.ToString("R");
```

Comment:
This will output 17 digits I think, if we change things we should be as close as what `double.ToString()` does as possible.

Reply:
it returns 15 digits ... which is identical to double.ToString()